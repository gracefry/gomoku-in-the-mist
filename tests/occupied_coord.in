place J1
who
place J1
who
history
term
