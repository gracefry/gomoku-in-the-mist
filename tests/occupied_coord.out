W
Occupied coordinate
W
J1
